spec/requests/api/v1/cocktails/rated_cocktails_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'rails_helper'
+
+RSpec.describe 'Rated Cocktails' do
+  before :all do
+    @user1 = create(:user)
+    @user2 = create(:user)
+
+    @cocktails1 = create_list(:cocktail, 3)
+    @cocktails1.each do |cocktail|
+      Rating.create!(user: @user1, cocktail: cocktail, stars: 1)
+    end
+
+    @cocktails2 = create_list(:cocktail, 3)
+    @cocktails2.each do |cocktail|
+      Rating.create!(user: @user2, cocktail: cocktail, stars: 1)
+    end
+  end
+
+  it 'returns a list of rated cocktails for the user' do
```

Comment:
Should we add some sad path testing here for if the user doesn't have rated cocktails to make sure it's returning what we expect?

Reply:
added a test to make sure it sends an empty array in no cocktails are rated